spec/spec_helper.rb

```diff
@@ -10,6 +10,8 @@
 end
 
 RSpec.configure do |config|
+  config.example_status_persistence_file_path = '.rspec_results'
```

Comment:
I added this so I could use `--next-failure` and `--only-failures` when running `rspec`, which I use a lot.